modules/security-aai/src/main/java/org/opencastproject/security/aai/ConfigurableLoginHandler.java

```diff
@@ -425,6 +465,14 @@ private String extractEmail(HttpServletRequest request) {
     if (StringUtils.equals(id, bootstrapUserId)) {
       roles.add(new JpaRole(GLOBAL_ADMIN_ROLE, organization));
     }
+	if (headerAffiliation != null) {
+      String affiliation = request.getHeader(headerAffiliation);
+	  List<String> affiliations = Arrays.asList(affiliation.split(";"));
+	  for (String eachAffiliation : affiliations) {
```

Comment:
No strong opinion, by eachAffiliation seems a somewhat weird choice for a variable name to me (e.g. read roleAffiliationPrefix + eachAffiliation). I suggest to use simply "affiliation" here.

Reply:
Since there already exists a variable called "affiliation", and "var" is too generic, I settled with "eachAffiliation".

Reply:
You could rename _affiliation_ to _affiliationHeader_, then the name _affiliation_ is free 

Anyway, no strong opinion here ;-)